### ISSUE TITLE
Feature request: workbox-cli to accept --config-file=... flag

### ISSUE THREAD

**Issue by bitterjug** [GoogleChrome/workbox#843]
**Library Affected**:
*workbox-cli*

**Browser & Platform**:
n/a

**Issue or Feature Request Description**:

`utils/getConfig.js` looks like it can take the config file name from one of the flags from meow. But `bin.js` doesn't configure meow with a config-file flag. If it did I could call from the command line and specify different config files for production and staging.

**Comment by gauntface** [comment]
Assigning to Jeff as he seems interested. Please bounce back if you disagree.

**Comment by jeffposnick** [comment]
I just gave this a try and it seems to be working as expected:

```sh
$ workbox generate:sw --config-file=test-config.js 
Using configuration from test-config.js.
```

I believe that's because `meow` will just parse anything that looks like a flag that it gets, without having to be specifically configured ahead of time.

Can you confirm that you're on the latest version of `workbox-cli` and try again?

**Comment by bitterjug** [comment]
@jeffposnick Thanks for a swift reply. You're right, if I point it at a file that _actually exists_ it works a lot better. So the problem I really had was that when I did 

    $ workbox inject:manifest --config-file=file_that_does_not_exist.js

It just showed me the help text. I kinda thought it might just work as it is, but I was silly when I tested it. I think there's scope for adding `--config-file` as a listed and documented argument, and maybe a kinder error message when the file doesn't exist? But I can get by with this.Thanks again.
